crates/server-primitives/Cargo.toml

```diff
@@ -9,14 +9,17 @@ license.workspace = true
 [dependencies]
 camino.workspace = true
 eyre.workspace = true
+libp2p.workspace = true
```

Comment:
you can use `libp2p_identity` directly here, since this is a primitive, it's best to keep it light and the `libp2p` crate will introduce a lot more than just identity